tests/hana_test.py

```diff
@@ -179,6 +179,14 @@ def test_update_conf_file(self):
             **{'sid': 'PRD', 'password': 'Qwerty1234', 'system_user_password': 'Qwerty1234'})
         self.assertTrue(filecmp.cmp(pwd+'/support/modified.conf', conf_file))
 
+    def test_update_hdb_pwd_file(self):
+        pwd = os.path.dirname(os.path.abspath(__file__))
+        shutil.copyfile(pwd+'/support/original.conf.xml', '/tmp/test.conf.xml')
+        hdb_pwd_file = hana.HanaInstance.update_hdb_pwd_file(
+            '/tmp/test.conf.xml', master_password='Master1234', 
+            sapadm_password = 'Adm1234', system_user_password='Qwerty1234')
```

Comment:
Please, remove the `spaces` after the `sapadm_password` parameter. They don't  follow pep8 rules